bin/project

```diff
@@ -32,6 +33,10 @@ _listprojects() {
     ls "$STATEDIR/projects" | grep -v .json
 }
 
+_deactivate_project() {
+    echo "" > "$STATEDIR/active-project"
```

Comment:
so what happens when attempting to run goto commands on a deactivated state?
How will `goto` behave?